src/app/clusters/fan-control-server/fan-control-server.cpp

```diff
@@ -318,7 +320,7 @@ void MatterFanControlClusterServerAttributeChangedCallback(const app::ConcreteAt
         break;
     }
     case PercentSetting::Id: {
-        DataModel::Nullable<uint8_t> percentSetting;
+        DataModel::Nullable<chip::Percent> percentSetting;
```

Comment:
```suggestion
        DataModel::Nullable<Percent> percentSetting;
```
Going to stop commenting on this; just fix them all, please.